Dockerfile

```diff
@@ -30,8 +30,9 @@ RUN echo 'extension=/usr/local/lib/php/extensions/no-debug-non-zts-20160303/imag
 COPY install-composer.sh /usr/local/bin/
 RUN install-composer.sh
 
-# Install Robo and add RoboFile.php
-RUN wget http://robo.li/robo.phar
+# Install Robo CI.
+# @TODO replace the following URL by http://robo.li/robo.phar when the Robo team fixes it.
+RUN wget https://github.com/consolidation/Robo/releases/download/1.1.5/robo.phar
```

Comment:
Actually, I prefer downloading from github because they at least have a TLS certificate, so this is a nice improvement. https://github.com/consolidation/Robo/issues/325 was a previous break on the old permalink too. Is there a permalink through github anywhere to the latest stable release? If not, I'm tempted to say we just keep this link and deal with upgrading the version (perhaps there's a robo self-update command?)